qtpy/QtGui.py

```diff
@@ -12,6 +12,28 @@
 
 if PYQT5:
     from PyQt5.QtGui import *
+    # Backport items moved to QtGui in Qt6
+    from PyQt5.QtWidgets import QAction, QActionGroup, QFileSystemModel, QShortcut, QUndoCommand
+
+    # Map missing/renamed methods
+    QColor.toTuple = lambda self: (self.red(), self.green(), self.blue(), self.alpha())
+    QColor.isValidColorName = QColor.isValidColor
+    QColor.fromString = lambda name: QColor(name)
+    QMouseEvent.position = lambda *args: QMouseEvent.pos(*args).toPointF()
+    if not hasattr(QFontMetrics, 'horizontalAdvance'):
+        QFontMetrics.horizontalAdvance = lambda self, *args, **kwargs: self.width(*args, **kwargs)
+    if not hasattr(QFontMetricsF, 'horizontalAdvance'):
+        QFontMetricsF.horizontalAdvance = lambda self, *args, **kwargs: self.width(*args, **kwargs)
+
+    # Fix enums in PyQt5 5.9.*
+    from .enums_compat import demote_enums
+    from PyQt5.QtCore import QT_VERSION_STR as __version__
+    if PYQT5 and __version__.startswith('5.9.'):
+        from PyQt5 import QtGui
+        demote_enums(QtGui)
+        del QtGui
+    del __version__, demote_enums
```

Comment:
```suggestion
    from PyQt5.QtCore import QT_VERSION_STR as __version__
    if __version__.startswith('5.9.'):
        from .enums_compat import demote_enums
        from PyQt5 import QtGui
        demote_enums(QtGui)
        del QtGui, demote_enums
    del __version__
```

* Move `.enums_compt` import and deletion inside the block, so it's only executed if necessary
* Remove redundant `PYQT5` check (since we're already inside the `PYQT5` block)